pxf/HDFSFileDataPXF.html.md.erb

```diff
@@ -2,388 +2,282 @@
+HDFS-plug-in-specific keywords and values used in the [CREATE EXTERNAL TABLE](../reference/sql/CREATE-EXTERNAL-TABLE.html) call are described in the table below.
+
+| Keyword  | Value |
+|-------|-------------------------------------|
+| \<host\>[:\<port\>]    | The HDFS NameNode and port. |
+| \<path-to-hdfs-file\>    | The path to the file in the HDFS data store. |
+| PROFILE    | The `PROFILE` keyword must specify one of the values `HdfsTextSimple`, `HdfsTextMulti`, or `Avro`. |
+| \<custom-option\>  | \<custom-option\> is profile-specific. Profile-specific options are discussed in the relevant profile topic later in this section.|
+| FORMAT 'TEXT' | Use '`TEXT`' `FORMAT` with the `HdfsTextSimple` profile when \<path-to-hdfs-file\> references a plain text delimited file.  |
+| FORMAT 'CSV' | Use '`CSV`' `FORMAT` with `HdfsTextSimple` and `HdfsTextMulti` profiles when \<path-to-hdfs-file\> references a comma-separated value file.  |
+| FORMAT 'CUSTOM' | Use the`CUSTOM` `FORMAT` with  the `Avro` profile. The `Avro` '`CUSTOM`' `FORMAT` supports only the built-in `(formatter='pxfwritable_import')` \<formatting-property\> |
+ \<formatting-properties\>    | \<formatting-properties\> are profile-specific. Profile-specific formatting options are discussed in the relevant profile topic later in this section. |
+
+*Note*: When creating PXF external tables, you cannot use the `HEADER` option in your `FORMAT` specification.
 
-## <a id="recordkeyinkey-valuefileformats"></a>Using a Record Key with Key-Value File Formats
+## <a id="profile_hdfstextsimple"></a>HdfsTextSimple Profile
 
-For sequence file and other file formats that store rows in a key-value format, the key value can be accessed through HAWQ by using the saved keyword '`recordkey`' as a field name.
+Use the `HdfsTextSimple` profile when reading plain text delimited or .csv files where each row is a single record.
 
-The field type must correspond to the key type, much as the other fields must match the HDFS data. 
+\<formatting-properties\> supported by the `HdfsTextSimple` profile include:
 
-WritableResolver supports read and write of recordkey, which can be of the following Writable Hadoop types:
+| Keyword  | Value |
+|-------|-------------------------------------|
+| delimiter    | The delimiter character in the file. Default value is a comma `,`.|
 
--   BooleanWritable
--   ByteWritable
--   DoubleWritable
--   FloatWritable
--   IntWritable
--   LongWritable
--   Text
 
-If the `recordkey` field is not defined, the key is ignored in read, and a default value (segment id as LongWritable) is written in write.
+The following SQL call uses the PXF `HdfsTextSimple` profile to create a queryable HAWQ external table from the `pxf_hdfs_simple.txt` file you created and added to HDFS in an earlier section:
```

Comment:
Let's not assume they've reached this topic by reading/completing the previous sections.  I think it would be better to roll the sample file creation steps directly into this procedure (making them optional).  That way readers can see exactly the correlation between the file contents, external table statement, and query output.  It's best to call this out in a new "Example" subsection with numbered steps.